### PR TITLE
Remove branding type from set email branding page

### PR DESCRIPTION
**Pull request by tombye**
Updating the 'Set email branding' page, from service settings, so it uses the new email_branding model added in https://github.com/alphagov/notifications-admin/pull/2249.

For a task in this story: https://www.pivotaltracker.com/story/show/159986276

### Set email branding page

|Before|After|
|---|---|
|old_set_email_branding|newer_set_email_branding|

Includes changes to the email preview page (iframe'd into the 'Set email branding' page) and the brand request page. They both needed changes so they accessed the brand type from the new email_branding model.

### CODE REVIEW

**Comment by yahoopete** [comment]
Does none mean GOV.UK only branding?  If so we should change the label to be clear about that....

**Comment by tombye** [comment]
@yahoopete we had a good look into that and none means it removes email branding from your service. That being the case, me and @quis reckon we show GOV.UK as what you get when you don't have a brand yet elsewhere (the branding request page) so it makes sense to do it here.

TLDR: Yep, now updated with a label of 'GOV.UK' (see new screenshot in the description).

**Comment by quis** [comment]
I get this error when I try to set a branding:

image
